cmd/geth/main.go

```diff
@@ -193,13 +193,6 @@ var (
 		utils.RPCGlobalTxFeeCap,
 	}
 
-	whisperFlags = []cli.Flag{
-		utils.WhisperEnabledFlag,
-		utils.WhisperMaxMessageSizeFlag,
-		utils.WhisperMinPOWFlag,
-		utils.WhisperRestrictConnectionBetweenLightClientsFlag,
-	}
```

Comment:
It might be better to leave the flags, but put them in 'DeprecatedFlags' and just issue warnings whenever any one of them is used. "The flag xxx has been deprecated; whisper has been moved to a separate repository (link)." Otherwise nodes which uses these flags will just fail to come up again after an update.